src/components/Api.js

```diff
@@ -0,0 +1,169 @@
+import { renderLoading } from "../utils/utils.js";
+
+import { api } from "../pages/index.js";
+import {
+  profileEditForm,
+  cardsEditForm,
+  propicForm,
+} from "../utils/constants.js";
+
+export class Api {
+  constructor(options) {
+    this._options = options;
+  }
+
+  loadUserInfo() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/users/me", {
```

Comment:
`https://around-api.en.tripleten-services.com/v1` is repeated in every request

According to the task you need to pass full `headers` and `baseUrl` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ .